cmd/util/ledger/reporters/account_reporter.go

```diff
@@ -64,7 +65,10 @@ func (r *AccountReporter) Report(payload []ledger.Payload) error {
 	defer rwm.Close()
 
 	l := migrations.NewView(payload)
-	st := state.NewState(l)
+	st := state.NewState(
+		l,
+		state.WithMaxInteractionSizeAllowed(math.MaxUint64),
+	)
```

Comment:
Had to increase the `MaxInteractionSizeAllowed` here and below.
cc: @janezpodhostnik